ntp.go

```diff
@@ -26,6 +26,7 @@ const (
 	broadcast
 	controlMessage
 	reservedPrivate
+	maxStratum uint8 = 16
```

Comment:
Please move this to another const declaration scope. I prefer not to mix it in with the mode constants.